Sources/Views/Gifts/GiftsLanding.swift

```diff
@@ -6,78 +6,7 @@ import PointFreeDependencies
 import PointFreeRouter
 import Prelude
 
-public func giftsLanding(
```

Comment:
most of this file can be deleted, but there are still a few things that are used in other gifts-related pages. once we convert those we can delete all of this.